### PR TITLE
feat: implement v2 SystemManagementClient

### LINKED ISSUE #622: [v2] Create client library for sys-mgmt-agent

### PR DESCRIPTION
**Pull request by hahattan**
Signed-off-by: Chris Hung <chris@iotechsys.com>

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/go-mod-core-contracts/blob/master/.github/Contributing.md.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## New Imports

- [ ] Yes
- [x] No

## Specific Instructions
Are there any specific instructions or things that should be known prior to reviewing?

## Other information

### CODE REVIEW

**Review by cloudxxx8 — APPROVED** [review]
LGTM

File: v2/clients/http/system.go [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+}
+
+func NewSystemManagementClient(baseUrl string) interfaces.SystemManagementClient {
+	return &SystemManagementClient{
+		baseUrl: baseUrl,
+	}
+}
+
+func (smc *SystemManagementClient) GetHealth(ctx context.Context, services []string) (res responses.HealthResponse, err errors.EdgeX) {
+	urlPath := path.Join(v2.ApiBase, "health", strings.Join(services, v2.CommaSeparator))
```
Comment (lenny-intel) [inline v2/clients/http/system.go:33]: These need to change to using query parameter for services list per Core WG discussion on 5/27/21

**Review by lenny-intel — CHANGES_REQUESTED** [review]

**Comment by hahattan** [comment]
change in this PR is now aligned with https://github.com/edgexfoundry/edgex-go/pull/3527

**Review by lenny-intel — APPROVED** [review]
LGTM

**Pull request merged**